Main.java

```diff
@@ -3,19 +3,37 @@ public static void main(String[] args) {
    // kindly use appropriate data types for your declaration
    // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor is to pay and assign values to them
    
+	
+	int amountPaidPerDay =1000;
+	int amountPaid = 3000;
+	int DefaultedDays = 4;
+   
    //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
+  int noOfdaysPaidfor ;
+  int AmounttoPay;
+  int daystoPayleft ;
+  int amounttocomplete ;
 
```

Comment:
Good job but your amount should have been in float, and use camelCase to edit variables